ssh/tailssh/tailssh.go

```diff
@@ -96,16 +96,17 @@ type server struct {
 
 	// mu protects activeSessions.
 	mu             sync.Mutex
-	activeSessions map[string]bool
+	activeSessions map[string]*tailcfg.SSHAction
 }
 
 var debugPolicyFile = envknob.String("TS_DEBUG_SSH_POLICY_FILE")
 
-// portForward reports whether the ctx should be allowed to port forward
+// localPortForward reports whether the ctx should be allowed to port forward
 // to the specified host and port.
 // TODO(bradfitz/maisem): should we have more checks on host/port?
-func (srv *server) portForward(ctx ssh.Context, destinationHost string, destinationPort uint32) bool {
-	return srv.isActiveSession(ctx)
+func (srv *server) localPortForward(ctx ssh.Context, destinationHost string, destinationPort uint32) bool {
```

Comment:
I don't think this method should be on `*server`.

I think we need a new type for an established session, that `launchProcess` or something earlier should return.